src/App.test.js

```diff
@@ -0,0 +1,13 @@
+import ReactDOM from "react-dom";
+import { render, screen } from "@testing-library/react";
+import App from "./App";
+
+it("renders without crashing", () => {
```

Comment:
change test-cases naming to correct convention, e.g. https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea